helpers.go

```diff
@@ -369,6 +370,12 @@ func InstallPostSymlinks(post *PostInstall, rootfsPath string) llb.RunOption {
 }
 
 func (s *Spec) GetSigner(targetKey string) (*PackageSigner, bool) {
+	if skipVal, ok := s.Args["DALEC_SKIP_SIGNING"]; ok {
+		if shouldSkip, err := strconv.ParseBool(skipVal); err == nil && shouldSkip {
+			return nil, false
```

Comment:
I noticed these bools are only true when we pass a non-nil signer back and false for every other case. Can we drop the bool?

Reply:
Sure